### PR TITLE
fix(VDialog): fix style of component to follow material spec

### LINKED ISSUE #7773: [Bug Report] VDialog Text left padding differs from md2 spec
### Versions and Environment
**Vuetify:** 2.0.0-beta.5
**Vue:** 2.6.10
**Browsers:** Chrome 76.0.3809.46
**OS:** Windows 10

### Steps to reproduce
Inspect `.v-card__text` element in dev tools and note padding value on the LH side.

### Expected Behavior
LH padding should be 24px

### Actual Behavior
LH padding is 16px

### Reproduction Link
<a href="https://codepen.io/anon/pen/gNBmKL" target="_blank">https://codepen.io/anon/pen/gNBmKL</a>

### Other comments
Dialog md2 spec found at https://material.io/design/components/dialogs.html#specs

vuetify-dialog-example
md2-dialog-example

### PR DESCRIPTION
**Pull request by avengerweb**
fix #7773

## How Has This Been Tested?
visually

## Markup:
VDialog spec:
<details>

```vue
<template>
  <v-container style="width: 250px">
    <div class="text-xs-center">
      <v-dialog
        v-model="dialog"
        width="500"
      >
        <template v-slot:activator="{ on }">
          <v-btn
            color="red lighten-2"
            dark
            v-on="on"
          >
            Click Me
          </v-btn>
        </template>

        <v-card>
          <v-card-title primary-title>
            Privacy Policy
          </v-card-title>

          <v-card-text>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
          </v-card-text>

          <v-divider></v-divider>

          <v-card-actions>
            <v-spacer></v-spacer>
            <v-btn
              color="primary"
              text
              @click="dialog = false"
            >
              I accept
            </v-btn>
          </v-card-actions>
        </v-card>
      </v-dialog>
    </div>
  </v-container>
</template>

<script>
export default {
  data: () => ({
    //
  })
}
</script>

```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:
- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.

### CODE REVIEW

File: packages/vuetify/src/components/VDialog/VDialog.sass [hunk 1]
```diff
@@ -18,6 +18,16 @@
   > *
     width: 100%
 
+  > .v-card
+    > .v-card__title
+      font-size: 1.25rem
```
Comment (johnleider) [inline packages/vuetify/src/components/VDialog/VDialog.sass]: These are the headings.h6 font size settings. Please update it to use them:

```sass
.v-card__title
  font-size: $headings.h6.size
  font-weight: $headings.h6.weight
```

etc

Comment (avengerweb) [reply packages/vuetify/src/components/VDialog/VDialog.sass]: Done!

**Comment by lorro** [comment]
Hi,

I just updated Vuetify and noticed all my cards that I had added in a dialog before have now a padding-top of 0 because of this commit. So the text sticks to the top now. Is that actually how it should be now? What should be the best/cleanest way to handle this? I could wrap just a div around the v-card as the css apparently applies to the v-card if it's a direct descendent of the v-dialog. But that's a bit hacky I think.

**Comment by avengerweb** [comment]
That affect only first card in dialog, please give link to reproduce

**Comment by lorro** [comment]
https://codesandbox.io/s/codesandbox-vue-vuetify-j39np

**Pull request merged**